### PR TITLE
🐛 Allow node scanning on rancher-provisioned controlplane and etcd nodes

### LINKED ISSUE #288: Node scanning for Rancher-provisioned clusters
I tried deploying the operator to a Rancher-provisioned cluster on a few virtual machines. The node scanning did not get scheduled on the master nodes because at the moment the DaemonSet tolerates only the `node-role.kubernetes.io/master` taint. However, in Rancher there are 2 different taints that are applied on the nodes depending on which components are running on them - namely:
- [node-role.kubernetes.io/etcd](https://rancher.com/docs/rke/latest/en/config-options/nodes/#etcd)
- [node-role.kubernetes.io/controlplane](https://rancher.com/docs/rke/latest/en/config-options/nodes/#controlplane)

If we would like the operator to support Rancher-provisioned clusters, we should make sure these labels are added to the list of tolerations.

### PR DESCRIPTION
**Pull request by imilchev**
Closes #288 

I tested it manually to verify these changes are sufficient to fix the issue. It seems to be working!

Signed-off-by: Ivan Milchev <ivan@mondoo.com>

### CODE REVIEW

**Review by joelddiaz — COMMENTED** [review]
This looks perfectly fine to me, but can we put comments near each of the tolerations that explains what environment each entry specifically enables?

**Comment by imilchev** [comment]
I added comments explaining the tolerations

File: controllers/nodes.go [hunk 1]
```diff
@@ -197,10 +197,26 @@ func (n *Nodes) daemonsetForMondoo() *appsv1.DaemonSet {
...
+						},
+						{
+							// Rancher etcd node
+							// https://rancher.com/docs/rke/latest/en/config-options/nodes/#etcd
+							Key:    "node-role.kubernetes.io/etcd",
+							Effect: corev1.TaintEffectNoExecute,
+							Value:  "true",
+						},
+						{
+							// Rancher controlplaen node
```
Comment (joelddiaz) [inline controllers/nodes.go:213]:
```suggestion
							// Rancher controlplane node
```

**Review by joelddiaz — APPROVED** [review]
👍

**Pull request merged**